docs/content/singleton-actors.md

```diff
@@ -0,0 +1,83 @@
+
+## Setting up Singleton Actors
+
+To properly set up Singleton Actors for your project, you need to:
+1. Register Singleton Actors with the `DefaultEditorSpatialGDK.ini` file
+1. Add the generated components to the UnrealWorker worker configuration file
+
+### Registering Singleton Actors
+
+You need to specify Singleton Actors in `DefaultEditorSpatialGDK.ini` before creating the snapshot and running SpatialOS.
+
+Using the [Unreal GDK Starter Project](https://github.com/improbable/UnrealGDKStarterProject) as an example, to do this:
+1. Locate the `DefaultEditorSpatialGDK.ini` file in the Starter Project repository which you cloned during installation and setup - it's located at `Game/Config/DefaultEditorSpatialGDK.ini`.
+2. Open the file in your editor and add your Singleton Actors as shown in the file snippet below.
+
+### Example file snippet
+
+In the snippet below, your game is `ExampleGame` and the Singleton Actor class you want to add is `ExampleGameGameState`
+
+```
+[SnapshotGenerator.SingletonActorClasses]
+ExampleGameGameState=
+
+[InteropCodeGen.ClassesToGenerate]
+;ClassName=include_path.h
+;Leave empty if no includes required.
+
+;ExampleGameGameState
+ExampleGameGameState=ExampleGameGameState.h
+ExampleGameGameState=IncludePath/MyClassDependency.h
+```
+
+### Example file description
+
+For an explanation of the `InteropCodeGen.ClassesToGenerate` section, see the [Interop Code Generator](./interop.md) documentation.
+
+`SnapshotGenerator.SingletonActorClasses`
+
+This section is for specifying the Singleton Actor classes the GSM will manage. In this case we specified one class, `ExampleGameGameState` which is the `GameState` class for `ExampleGame`.
+
+For each Singleton Actor class, you also need to generate type bindings, so you have to add the class to the `InteropCodeGen.ClassesToGenerate` section.
+
+### Streaming queries
+
+To make sure all servers check out Singleton Actor entities, you need to configure the worker to have [streaming queries](https://docs.improbable.io/reference/13.1/shared/worker-configuration/bridge-config#streaming-queries) for each Singleton Actor’s components.
+
+In our example with `ExampleGameGameState`, the Interop Code Generator creates a schema component called `ExampleGameGameStateMultiClientRepData`. You need to add this as a streaming query to the worker configuration file. You can find this in `spatial/workers/unreal/spatialos.UnrealWorker.worker.json`. 
```

Comment:
Just realised this reads a bit strangely now; how about:

> You need to add this as a streaming query to the worker configuration file (`spatial/workers/unreal/spatialos.UnrealWorker.worker.json`).

instead of "You can find this in..." because that _is_ the worker config file, right? 

Other than this, all good!